utils/dependency_management.py

```diff
@@ -30,7 +30,7 @@
     'graphviz': 'python-graphviz',
 }
 
-CONDA_IGNORE = ['pyblake2', r'.*python_version == \"3\.5\"']
+CONDA_IGNORE = ['pyblake2']
```

Comment:
didn't you remove pyblake2 from the dependency list?

Reply:
I guess, it is a backport from `hashlib.blake2b` which become part of stdlib from Python 3.6. So this would then also apply to conda, wouldn't it? Do you want me to also remove the `CONDA_IGNORE` variable and associated code, or we just leave it as an empty list.

Reply:
> Do you want me to also remove the CONDA_IGNORE variable and associated code, or we just leave it as an empty list.

Up to you - it's difficult for me to guess how likely it is that we will need such exceptions again.

If you decide to leave it in for the moment, perhaps add a comment explaining what it does and setting an ultimatum in 6? months after which it can be removed

Reply:
@csadorf I think this is your code. What do you prefer? I leave it in for now?

Reply:
I'd prefer to leave it in.